src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+
+    @Override
```

Comment:
```suggestion
public class Excavator extends Machine {
    @Override
```
старайся не розтягувати код і не залишати зайвих пустих ліній